### DISCUSSION TITLE
Discuss and share anything at "Discussions" forum

### DISCUSSION THREAD

**Discussion by yuweihao** [yuweihao/MambaOut#182] [General]
Hi everyone, the "Discussions" forum may be more appropriate for discussing and sharing anything.

I really respect Kobe. He may not have the best talent, but he achieved greatness with the Mamba Mentality.

Besides, I deeply understand that research is exhausting. I recently watched an interesting Korean drama, ‘Queen of Tears,’ (眼泪女王 in Chinese) and I’d like to share it with everyone, hoping it can help you relax outside of work.

**Comment by poult-lab** [comment]
Hello authors, I also watched Queen of Tears. I know the main male character, Kim, is now in big trouble. By the way, do you like Squid Game? Netflix will also release many Korean dramas in the fall of 2025.
